env-builder-data/build/script/packet/glib-2.69.3.sh

```diff
@@ -41,7 +38,7 @@ pkinstall() {
 #        cat <<EOT >> ${INSTALL_PACKET_DIR}/bin/glib-compile-resources
 ##!/bin/bash
 #
-#glib-compile-resources "$@"
```

Comment:
How do you compile resources if both are commented out?

Reply:
In this file - "[env-builder-data/build/script/packet/gtk-3.22.12.sh](https://github.com/morevnaproject/morevna-builds/pull/79/files#diff-8d3a4f602d5c45c5fdc483e9b3afd85076d10fe5b6aaf8bedcd4883a0b89efca)" - I add glib-2.69.3 as native dependency. So, the builder compiles glib and provides native binary for "glib-compile-resources".